during_the_workshop.md

```diff
@@ -216,7 +220,31 @@ Hint: to rebuild your containers you may need to call docker-compose with the `-
 $ docker-compose up --build
 ```
 
-## Part 7: nginx (stretch)
+## Part 7: Improve the cliapp image (stretch goal)
+The cliapp image currently generates three datasets and then exits. You need to manually run it again in order to get more recent data. Let's update it so that it runs a scheduled job instead.
+
+First, write a crontab file which runs the `run.sh` file every five minutes (or every minute for faster testing) and redirects the output to `/var/log/cron.log`. Note that the cronjob will not run from the Dockerfile's `WORKDIR`.
+<details><summary>Expand to see the cron job</summary>`* * * * * /opt/chimera/bin/run.sh >> /var/log/cron.log 2>&1`</details>
+
+Then update the Dockerfile to accomplish the following steps:
+1) Install cron with `apt-get`
+2) Create an empty log file at `/var/log/cron.log` that the job can send its output to
+3) `COPY` the crontab file into the image.
+4) Load the crontab file by running `crontab your_crontab_file`
+5) Write an `ENTRYPOINT` that starts `cron` and doesn't just exit immediately. Otherwise the container will exit immediately, without running scheduled jobs. <details><summary>Answer</summary>You can either run cron in the foreground with `cron -f` or perhaps more usefully you could print the logs with tail, if your cronjob is directing its output to a log file, e.g.: `cron && tail -f /var/log/cron.log`. This will save you having to dig into the container to debug issues</details>
+
+You will probably see some error messages at this point (after the job has had a chance to run) that you need to address.
```

Comment:
I ran into an issue here with my crontab file having the wrong line endings - worth flagging? (I've raised a suggestion above)